app/models/user.rb

```diff
@@ -23,4 +23,8 @@ def self.new_with_session(params, session)
       end
     end
   end
-end
+  
+  def admin?
+    self.kind == "admin" ? true : false
```

Comment:
boolean operator already returns booolean
```suggestion
    self.kind == "admin"
```

Reply:
Let me correct this.